test/functions_math.jl

```diff
@@ -208,6 +208,13 @@ end
     @test inv(nda) * nda ≈ NamedDimsArray{(:b, :b)}([1.0 0; 0 1])
 end
 
+@testset "diff" begin
+    arr = [1.0 2; 3 4]
+    nda = NamedDimsArray{(:a, :b)}(arr)
+    @test unname(diff(nda; dims=:a)) == diff(arr; dims=1)
```

Comment:
```suggestion
    @test diff(nda; dims=:a) == diff(arr; dims=1)
```

I think this should work?